src/Security/SecurityControllerBuilder.php

```diff
@@ -55,4 +63,19 @@ public function addLoginMethod(ClassSourceManipulator $manipulator)
         );
         $manipulator->addMethodBuilder($loginMethodBuilder);
     }
+
+    public function addLogoutMethod(ClassSourceManipulator $manipulator)
+    {
+        $logoutMethodBuilder = $manipulator->createMethodBuilder('logout', null, false, ['@Route("/logout", name="app_logout" , methods={"GET"})']);
+
+        $manipulator->addUseStatementIfNecessary(Response::class);
+        $manipulator->addUseStatementIfNecessary(Route::class);
+        $manipulator->addUseStatementIfNecessary(AuthenticationUtils::class);
+        $manipulator->addMethodBody($logoutMethodBuilder, <<<'CODE'
+<?php
+// controller can be blank: it will never be executed!
```

Comment:
Let's throw an exception here with a similar message inside:

```
throw new \Exception('This method can be blank - it will be intercepted by the logout key on your firewall');
```